test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs

```diff
@@ -48,6 +48,41 @@ public async Task CheckNonOptionalSettingsThrowsOnMissingAzureWebJobsStorage()
                 $"This is required for all triggers other than {string.Join(", ", Constants.TriggersWithoutStorage)}.");
         }
 
+        [Fact]
+        public async Task CheckNonOptionalSettingsDoesntThrowMissingStorageUsingManagedIdentity()
+        {
+
+            Skip.IfNot(RuntimeInformation.IsOSPlatform(OSPlatform.Windows),
+                reason: "Environment.CurrentDirectory throws in linux in test cases for some reason. Revisit this once we figure out why it's failing");
+            var fileSystem = GetFakeFileSystem(new[]
+                {
+                ("x:\\folder1", "{'bindings': [{'type': 'blobTrigger'}]}"),
+                ("x:\\folder2", "{'bindings': [{'type': 'httpTrigger'}]}")
+            });
+
+            var secrets = new Dictionary<string, string>()
+            {
+                { "AzureWebJobsStorage:blobServiceUri", "myuri" },
+                { "AzureWebJobsStorage:queueServiceUri", "queueuri" }
```

Comment:
Can you make this one use `AzureWebJobsStorage__queueServiceUri` to check that block of code?